spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCallStatementParser.java

```diff
@@ -76,10 +73,9 @@ public static void stopSpark() {
   public void testCallWithPositionalArgs() throws ParseException {
     CallStatement call =
         (CallStatement) parser.parsePlan("CALL c.n.func(1, '2', 3L, true, 1.0D, 9.0e1, 900e-1BD)");
-    Assert.assertEquals(
-        ImmutableList.of("c", "n", "func"), JavaConverters.seqAsJavaList(call.name()));
+    assertThat(seqAsJavaList(call.name())).hasSameElementsAs(ImmutableList.of("c", "n", "func"));
```

Comment:
```suggestion
    assertThat(seqAsJavaList(call.name())).containsExactly("c", "n", "func");
```

please also update the other places